src/ResourceManager/LogicApp/Commands.LogicApp.Test/ScenarioTests/Common.ps1

```diff
@@ -73,6 +73,17 @@ function TestSetup-CreateResourceGroup
 
 <#
 .SYNOPSIS
+Creates named resource group to use in tests
+#>
+function TestSetup-CreateNamedResourceGroup([string]$resourceGroupName)
+{    
+    $resourceGroup = New-AzureRmResourceGroup -Name $resourceGroupName -location "brazilsouth" -Force
```

Comment:
Please do not hard code locations.  use the Get-location or get-provider cmdlets to determine an available location (it's fine to prefer brazilsouth if available.  This makes it easier to maintain the tests, and to run them against different environments